onramp_compassion/models/sponsorship_correspondence.py

```diff
@@ -100,6 +119,9 @@ def _commkit_update(self, data, message_id=None):
         else:
             commkit = self.with_context(from_onramp=True).create(commkit_vals)
 
+        # Send email to sponsor
+        commkit.send_email()

```

Comment:
I think that you forgot to move this call in switzerland repository, don't you ?